### PR TITLE
bumping up babel to version 7

### PR DESCRIPTION
**Pull request by mspoerer**
Introduction of babel 7

### CODE REVIEW

**Comment by Hypnosphi** [comment]
Shouldn't we wait until it becomes stable?

File: src/index.js [hunk 1]
```diff
@@ -100,7 +102,7 @@ function isExported(path, className, t){
     if(path.node.type === 'ExportNamedDeclaration' &&
        path.node.specifiers &&
        path.node.specifiers.length) {
-      return className === path.node.specifiers[0].exported.name;
+      return className === path.node.specifiers[0].local.name;
```
Comment (mspoerer) [inline src/index.js]: that should fix 'export default Component'

**Comment by mspoerer** [comment]
I totally agree. But I had to bump up the version for my project anyway. So i though i would let you guys participate. It also should contains a fix for [#33](https://github.com/storybooks/babel-plugin-react-docgen/issues/33) (at least for the part 'export default Component' in conjunction with babel7). I didn't head the time to cross-check it with babel 6.

**Comment by danielduan** [comment]
Hey @mspoerer, thanks for the PR. I will keep this open for the stable build. Feel free to give me another ping whenever the stable is released and you have updated the versions in this. Thanks!

**Comment by danielduan** [comment]
superceded by #57

**Pull request closed without merging**